components/ui/Sidebar.tsx

```
@@ -0,0 +1,133 @@
+                {/* Folder Header */}
+                <div className="px-2 py-1 text-xs font-semibold text-[#A08465] uppercase">
+                  {folder}
+                </div>
+
+                {/* Documents in Folder */}
+                {docs.map((doc) => (
+                  <button
+                    key={doc.id}
+                    onClick={() => onDocumentClick(doc.id)}
+                    className="w-full text-left px-3 py-2 rounded-md text-[#E8DCC4] hover:bg-[#3D3426] transition-colors flex items-center gap-2"
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)